src/coreclr/jit/codegen.h

```diff
@@ -60,9 +60,6 @@ class CodeGen final : public CodeGenInterface
     // Bit mask used in zeroing the 3rd element of a SIMD12
     CORINFO_FIELD_HANDLE zroSimd12Elm3;
 
-    // Bit mask used in U8 -> double conversion to adjust the result.
-    CORINFO_FIELD_HANDLE u8ToDblBitmask;
```

Comment:
Unrelated, but I don't think any of these handles are actually needed anymore.

The method local constant APIs already browse the constant table for matching bit patterns to avoid duplicating them.

Reply:
Good point, will delete them once current CI run finishes